src/pages/home/_lazy-load.js

```diff
@@ -1,5 +1,6 @@
 import Loadable from '@loadable/component'
 
+export const MarketsFold = Loadable(() => import('./_markets_fold'))
```

Comment:
This section would probably still be visible on desktop initial loads so probably best to exclude it from the lazy load.

Reply:
Yeah for sure 👍🏼